### DISCUSSION TITLE
How to seed a default Account (and associated User) ?

### DISCUSSION THREAD

**Discussion by bdavidxyz** [janko/rodauth-rails#88] [General]
How to create a default account, using the seed.rb file ?

**Comment by janko** [comment]
The `Rodauth::Rails::Model` mixin provides a [password attribute](https://github.com/janko/rodauth-rails#password-attribute), so you'd create the account in a similar way as when using Devise or has_secure_password:

```rb
Account.create!(email: "user@example.com", password: "secret", status: "verified")
```
